// Copyright 2023 gRPC authors.

syntax = "proto3";

package grpc.testing;

message ResourceQuota {}

message FuzzingChannelArg {
    string key = 1;
    oneof value {
        string str = 2;
        int64 i = 3;
        ResourceQuota resource_quota = 4;
    }
}

message FuzzingChannelArgs {
    repeated FuzzingChannelArg args = 1;
}
